src/components/ListTemplate/ColumnTemplates/DefaultColumn.js

```diff
@@ -2,8 +2,12 @@ import React, { memo } from 'react';
 import HighLight from 'react-highlighter';
 import PropTypes from 'prop-types';
 
+import { Button } from '@folio/stripes/components';
+
```

Comment:
Remove the blank line, please. This import and import below belong to one group - imports of stripes components